scripts/savedjobs_notify.py

```diff
@@ -0,0 +1,49 @@
+import sys, os, django
+
+sys.path.append("../nycCivilServiceJobs")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "nycCivilServiceJobs.settings")
+django.setup()
+
+from jobs.models import job_record, UserSavedJob
+from django.core import mail
+from django.template.loader import render_to_string
+import datetime
+
+
+def send_email(to, subject, content):
+    mail.send_mail(subject, content, None, [to], html_message=content)
+
+
+def notify_jobs():
+    queryset = UserSavedJob.objects.filter(
+        job__post_until__gte=datetime.date.today(),
+        job__post_until__lte=datetime.date.today() + datetime.timedelta(days=2),
+    )
+
+    users = queryset.values_list("user", flat=True).distinct()
+    email = ""
+    for user in users:
+        user_sub = queryset.filter(user=user)
+
+        job_list = []
+
+        for obj in user_sub:
+            job_id = obj.job.job_id
+
+            job_list.extend(
```

Comment:
If you filter job_record by job_id you might get multiple jobs because there can be more than one jobs with same job_id. Instead, you may wanna use primary key (id) or you can directly use job_list.extend(obj.job) since obj.job would give the job record.

Reply:
Fixed in [commit](https://github.com/gcivil-nyu-org/nycCivilServiceJobs/commit/fcbdfe28bdd3399c29d03f45a9a476a992ddf7ea)

Reply:
Thanks this has been resolved